src/Hero6/Engine/Utilities/Settings/UserSettings.Desktop.cs

```diff
@@ -58,7 +62,12 @@ public int GameStartedCount
             private set { data.GameStartedCount = value; }
         }
 
-        public void Save() => file.WriteAllText(Filename, JsonConvert.SerializeObject(data));
+        public void Save()
+        {
+            directory.CreateDirectory(Game.UserFilesDir); // If dir already exists, does nothing
```

Comment:
I feel this is implicit what it does and why it does it so the comment is unnecessary. Can you remove it?

Reply:
Not necessarily. Though convenient, I've always felt it was non-intuitive that CreateDirectory should not throw an error if the Dir already exists. I've modified my comment to better clarify the purpose for the comment, does that help?

Reply:
Meh. Agree to disagree. It's really a nitpick so let's just move on and get this PR merged.